mantidproject/mantid
# Revised MVP tutorial.

I've revised the MVP tutorial to be more user-friendly and made the steps to building the complete GUI more logically laid out. 
I've moved the multiple views and matplotlib pages to the end of the tutorial and made a final exercise to bring it all together.

**To test:**

1. Build dev-docs.
1. Open the tutorial in [BuildDirectory]/devdocs/HTML/MVPTutorial
1. Check that the new layout legible and fit for purpose. 
1. Check that the scripts given in CompleteGUI.html run correctly.

*There is no associated issue.*

*This does not require release notes* because it is a minor documentation change.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
